engine/src/main/resources/org/camunda/bpm/engine/db/create/activiti.cockroachdb.create.history.sql

```diff
@@ -35,6 +35,7 @@ create table ACT_HI_PROCINST (
     DELETE_REASON_ varchar(4000),
     TENANT_ID_ varchar(64),
     STATE_ varchar(255),
+    RESTARTED_PROC_INST_ID_ varchar(64),
```

Comment:
See [point A](https://github.com/camunda/camunda-bpm-platform/pull/4420#issuecomment-2337712471) in the ticket thread